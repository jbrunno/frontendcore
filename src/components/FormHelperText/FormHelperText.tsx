import React from 'react';
import {
  FormHelperText as MuiFormHelperText,
  FormHelperTextProps as MuiFormHelperTextProps,
} from '@mui/material';

export type FormHelperTextProps = MuiFormHelperTextProps;

export function FormHelperText(props: FormHelperTextProps) {
  return <MuiFormHelperText {...props} />;
}

export default FormHelperText;
