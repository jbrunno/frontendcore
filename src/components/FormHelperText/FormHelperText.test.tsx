import React from 'react';
import { render } from '@test-utils/component-testing';
import { FormHelperText } from './FormHelperText';

test('renders the FormHelperText', () => {
  const { container } = render(<FormHelperText />);
  expect(container).toMatchSnapshot();
});
