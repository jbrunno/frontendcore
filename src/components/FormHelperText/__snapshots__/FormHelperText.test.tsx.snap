// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the FormHelperText 1`] = `
.emotion-0 {
  color: #3B3B4199;
  font-weight: 400;
  font-size: 0.75rem;
  line-height: 1.125rem;
  letter-spacing: 0.0106rem;
  font-family: Lato,sans-serif;
  text-align: left;
  margin-top: 3px;
  margin-right: 0;
  margin-bottom: 0;
  margin-left: 0;
}

.emotion-0.Mui-disabled {
  color: #3B3B4161;
}

.emotion-0.Mui-error {
  color: #F44336;
}

<div>
  <p
    class="MuiFormHelperText-root emotion-0"
  />
</div>
`;
